internal/olm/operator/registry/configmap/deployment_test.go

```diff
@@ -0,0 +1,266 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package configmap
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/operator-framework/operator-sdk/internal/util/k8sutil"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Deployment", func() {
+
+	Describe("getRegistryServerName", func() {
+		It("should return the formatted servername", func() {
+			name := k8sutil.FormatOperatorNameDNS1123("pkgName")
+			name = fmt.Sprintf("%s-registry-server", name)
+
+			Expect(getRegistryServerName("pkgName")).Should(Equal(name))
```

Comment:
Remove these 3 lines. Because line 36 is the same test. I don't want to use the implementation of the method in the test if we don't have to.